assets/js/renderer.js

```diff
@@ -408,6 +408,21 @@ $(document).ready(function () {
         );
     });
 
+    // Open a SQLite Database File -- custom menu
+    ipcRenderer.on('open-SQLite', function () {
+        dialog.showOpenDialog({
```

Comment:
@johnmhoran Could you refactor the duplicated code in the `ipcRenderer` and the `openDBFileButton` click event into a separate function, and then have them both call that function?

You could also change the `openDBFileButton` to `openSQLiteButton` so that we are consistent.